make-based/include/common_functions

```diff
@@ -38,6 +50,44 @@ setup_all_libs()
     done
 }
 
+reset()
+{
+    if test ! -d "$app"; then
+        echo "$app folder doesn't exist. Did you run '$0 setup'?" 1>&2
+        exit 1
+    fi
+    if test ! -d "$uk"; then
+        echo "$uk folder doesn't exist. Did you run '$0 setup'?" 1>&2
+        exit 1
+    fi
+    for l in $all_libs; do
+        if test ! -d "$l"; then
+            echo "$l folder doesn't exist. Did you run '$0 setup'?" 1>&2
+            exit 1
```

Comment:
```suggestion
            exit 1
```
This will go through all the libraries, including the ones that were not pulled by the `setup`.
We should either not exit here, just state that we did not find the library, or go through the application libraries (`$app_libs`), not through `$all_libs`.